presto-sqlserver/src/test/java/io/prestosql/plugin/sqlserver/SqlServerQueryRunner.java

```diff
@@ -60,7 +60,7 @@ public static QueryRunner createSqlServerQueryRunner(TestingSqlServer testingSql
             queryRunner.installPlugin(new TpchPlugin());
             queryRunner.createCatalog("tpch", "tpch");
 
-            connectorProperties = new HashMap<>(ImmutableMap.copyOf(connectorProperties));
```

Comment:
it was verifying there are no nulls

Reply:
I had the same thought initially, but assumed it was leftover from a refactoring. In the case that it's intentional, let's add a comment.

Reply:
Or make the verification explicit:

```java
checkArgument(!map.containsKey(null) && !map.containsValue(null), ...);
```